babel.config.js

```diff
@@ -25,7 +25,7 @@ module.exports = function(api) {
 					modules: noModules ? false : 'auto',
 					exclude: ['@babel/plugin-transform-typeof-symbol'],
 					targets: {
-						browsers: ['last 2 versions', 'IE >= 9']
+						browsers: ['last 2 versions', 'not IE 11']
```

Comment:
Apparently `not IE 11` includes IE 10, blackberry 10 and other relics from the past: https://browserslist.dev/?q=bGFzdCAyIHZlcnNpb25zLCBub3QgSUUgMTE%3D

Not sure what a good replacement value would be that only includes somewhat modern browsers. Maybe @developit knows more?

Reply:
Need to include 'not dead'. But this was the main reason I am thinking we might be better off using targets.esmodules.

Reply:
esbuild seems to fail when I do that 😅

Reply:
Probably a simplified browserslist implementation. This should work and is equivalent:

```
"Firefox>=60, chrome>=61, and_chr>=61, Safari>=10.1, iOS>=10.3, edge>=16, opera>=48, op_mob>=48, Samsung>=8.2, not dead"
```

Reply:
I don't think microbundle actually looks at this babel.config 😅

Reply:
correct - it uses `"browserslist"` from the package.json.